crates/autopilot/src/arguments.rs

```diff
@@ -153,9 +153,9 @@ pub struct Arguments {
     #[clap(long, env, action = clap::ArgAction::Set, default_value = "false")]
     pub enable_colocation: bool,
 
-    /// Driver base URLs.
+    /// A list of drivers in the following format: `<NAME>|<URL>,<NAME>|<URL>`
     #[clap(long, env, use_value_delimiter = true)]
-    pub drivers: Vec<Url>,
+    pub drivers: Vec<ExternalSolver>,
```

Comment:
nit: Either we don't consider Gnosis solvers external then this name doesn't capture all solvers. Or we consider them external then the `External` prefix is kind of redundant.
Maybe `DriverConfig`?